solr/solr-ref-guide/build.gradle

```diff
@@ -120,15 +120,15 @@ ext {
          path: 'html-site',
          desc: 'HTML Site for publishing to the Solr website',
          props: [
-            htmlSolrJavadocs: "https://lucene.apache.org/solr/${solrGuideVersionPath}_0/",
+            htmlSolrJavadocs: "https://solr.apache.org/${solrGuideVersionPath}_0/",
```

Comment:
Actually I chose `/docs/X_Y_Z` as the javadoc root. But due to `.htaccess` rule, `/X_Y_Z` also works, but is redirected.

Reply:
The Ref Guide only builds on minor releases (X_Y), not patch releases (X_Y_Z) so we only ever need to link to javadocs for 8_7_0, 8_8_0, etc. I think that's what you meant?

Reply:
I believe Jan meant that the path of the javadocs should be prefixed with `/docs/`, i.e. "https://solr.apache.org/docs/${solrGuideVersionPath}_0/"

Reply:
The `docs` is added automatically as far as I can tell. The attributes in the `.adoc` are used to construct a URL in the HTML file, which in this example becomes:

`https://solr.apache.org/9_0_0//core/org/apache/solr/handler/component/SuggestComponent.html` 

when one clicks on it and it goes through the javadocs redirect it comes out as:

`https://solr.apache.org/docs/9_0_0/core/org/apache/solr/handler/admin/ShowFileRequestHandler.html`

Reply:
Oh, okay, I see what you're saying. The example I have is the redirect Jan is talking about. But I don't see why it matters? There's already a redirect so let it redirect?

Reply:
I just thought we could advertise the /docs/ path as the one people should use.